sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/CxxSquidSensor.java

```diff
@@ -400,7 +401,7 @@ private Iterable<InputFile> getInputFiles(SensorContext context, CxxSquidConfigu
       // then analyze only the files contained in the db.
       var inputFilesInConfig = squidConfig.getFiles();
       var result = StreamSupport.stream(inputFiles.spliterator(), false)
-        .filter(f -> inputFilesInConfig.contains(Path.of(f.uri())))
+        .filter(f -> inputFilesInConfig.contains(Paths.get((Path.of(f.uri()).toString()).toLowerCase(Locale.getDefault()))))

```

Comment:
Depending on the operating system and SonarQube Database settings paths can be case-sensitive (also on case-insensitive operating systems). Think making it always lowercase does not work?